buildchain/buildchain/targets/package.py

```diff
@@ -349,7 +349,7 @@ def __init__(
     def deb(self) -> Path:
         """DEB path."""
         fmt = '{pkg.name}_{pkg.version}-{pkg.build_id}_{pkg.ARCH}.deb'
-        return self.rootdir/fmt.format(pkg=self)
+        return constants.PKG_DEB_ROOT/fmt.format(pkg=self)
```

Comment:
In the final staging area, this looks odd since we have every other `.deb` that was downloaded set inside its directory (so `_build/packages/metalk8s-bionic/*.deb` etc), but the built packages are set in `_build/packages`.

Could we make that `constants.PKG_DEB_ROOT / 'metalk8s-scality' / fmt.format(pkg=self)` ?

Reply:
It will be handled by the issue #1781